ethereum/eth/src/test/java/tech/pegasys/pantheon/ethereum/eth/manager/MockEthTask.java

```diff
@@ -19,11 +19,9 @@
 public class MockEthTask extends AbstractEthTask<Object> {
 
   private boolean executed = false;
-  private CountDownLatch countdown;
```

Comment:
If we are going to verify non-execution for forced shutdown we will need the unblockable tasks.

Reply:
Turns out we needed it for `EthSchedulerTest` anyway - it just didn't need the countdown method.